app/views/layouts/application.html.haml

```diff
@@ -12,9 +12,8 @@
         .row.row-offcanvas.row-offcanvas-left{style: "height: 100%"}
           #sidebar.col-sm-4.col-md-3.sidebar-offcanvas{:role => "navigation"}
             = render file: "shared/sidenav"
-          .col-md-9.col-sm-9.main
+          .col-md-9.col-sm-8.main
```

Comment:
To niestety nie rozwiązało problemu z rozjeżdżającym się layoutem. Poza tym powstaje niekonsekwencja w wyświetlaniu (tak mi się przynajmniej wydaje).